python/surf/devices/silabs/_Si5345Lite.py

```diff
@@ -28,6 +37,27 @@ def __init__(self,
             value        = "",
         ))
 
+        if self._useVars:
+
+            # Create 4 x 4K Blocks
```

Comment:
@slacrherbst I tried using this code (which is currently broken) as a template to port over some ASIC code from _rawWrite to this new method in rogue @ v5.5 but I got an error message that block memory size must be between 4 - 2048 bytes.  You would get that error message with this 4kB DataBlock declaration

Reply:
fixed in latest code that I pushed